modules/packages.nix

```diff
@@ -14,10 +14,6 @@
     gnupg
     wget
 
-    # Nix #
-    alejandra
-    statix
-
```

Comment:
Okay actually maybe add this back since I don't totally understand devshells yet :p